src/xrGame/xrGame.vcxproj

```diff
@@ -17,6 +17,7 @@
   <PropertyGroup Label="Globals">
     <ProjectGuid>{200652A6-043E-4634-8837-87983B3BD5E0}</ProjectGuid>
     <RootNamespace>xrGame</RootNamespace>
+    <WindowsTargetPlatformVersion>8.1</WindowsTargetPlatformVersion>
```

Comment:
Windows target version is unset for all the project files. This is not critical, but it's recommended to not change this.